cocotb/triggers.py

```diff
@@ -708,13 +733,15 @@ def on_done(ret):
 
 
 class ClockCycles(Waitable):
-    """
-    Fires after *num_cycles* transitions of *signal* from ``0`` to ``1``.
-    """
     def __init__(self, signal, num_cycles, rising=True):
         """
-        :param rising: If true, the default, count rising edges. Otherwise,
-            count falling edges
+        Fires after *num_cycles* transitions of *signal* from ``0`` to ``1``.
```

Comment:
I'd put this back in the class docstring

Reply:
Done.